cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -186,10 +186,46 @@ The following is the generic procedure for Major/Minor version upgrades:
 
 .. _cloudera-release-specific-upgrade-notes:
 
+.. rubric:: Upgrading CDAP 3.0 to 3.1 and Upgrading CDH 5.3 to 5.4
+
+**Background:** CDH 5.3 ships with Hbase 0.98 while CDH 5.4 supports HBase 1.0. We support
```

Comment:
"CDH 5.4 supports HBase 1.0":  "supports" makes it sound optional.  I suggest "includes" or repeat "ships".